tests/__snapshots__/index.test.ts.snap

```diff
@@ -1231,7 +1242,10 @@ export default class TypeEnum extends React.Component<Props> {
       foo: _pt.string.isRequired
     }), _pt.shape({
       bar: _pt.number
-    })])
+    })]),
+    typeRefEnumIntersection: _pt.oneOfType([_pt.oneOf([Baz.Bar]), _pt.oneOf([Qux.Bar])]),
```

Comment:
Should be `typeRefEnumIntersection: _pt.oneOf([1, 1])`. Should probably unique the values to check also.